### PR TITLE
Update to version 1.1.1

### PR DESCRIPTION
**Pull request by goanpeca**
@jakirkham, better?

### CODE REVIEW

**Comment by jakirkham** [comment]
Yep, this is better.

Not sure what was causing the failures before, but I know this feedstock is a little out of date. Could we please [re-render](https://github.com/conda-forge/conda-smithy#re-rendering-an-existing-feedstock) `conda-smithy` version `1.3.2`? If you had installed `conda-smithy` previously, you may need to update `conda-build-all` to version `1.0.0` as well.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
 package:
-  name: pydocstyle
+  name: {{ name }}
   version: {{ version }}
 
 source:
-  fn: pydocstyle-{{version}}.zip
-  url: https://pypi.io/packages/source/p/pydocstyle/pydocstyle-{{ version }}.zip
-  md5: {{ md5 }}
+  fn: {{ name }}-{{version}}.zip
```
Comment (jakirkham) [inline recipe/meta.yaml]: Maybe we should space out `version` a bit too.

Comment (goanpeca) [reply recipe/meta.yaml]: Right!

Comment (goanpeca) [reply recipe/meta.yaml]: Why is it failing  :-| ?

Comment (jakirkham) [reply recipe/meta.yaml]: I think the feedstock is out-of-date. If we re-render, the problems should be fixed. See this [comment](https://github.com/conda-forge/pydocstyle-feedstock/pull/3#issuecomment-253681318) for details.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -44,5 +46,4 @@ extra:
   recipe-maintainers:
     - goanpeca
     - jakirkham
-    - Nurdok
-
+    - Nurdok
```
Comment (jakirkham) [inline recipe/meta.yaml]: nit: Please keep a terminal newline.

File: recipe/meta.yaml (hunk 1 above)
Comment (goanpeca) [reply recipe/meta.yaml]: Done, thanks :-)

**Review by jakirkham — APPROVED** [review]

**Comment by goanpeca** [comment]
@jakirkham build have not progressed in like 12 hours... is this normal?

**Comment by jakirkham** [comment]
Yeah, the CIs have been a mess of late. Not sure what to say. 😞

I think we need to re-render with `1.3.3` to fix AppVeyor.

**Comment by goanpeca** [comment]
@jakirkham just re-rendered, lets see if it works

**Comment by goanpeca** [comment]
@jakirkham, all ci good again. Merge :-) ?

**Comment by jakirkham** [comment]
Thanks @goanpeca .

**Pull request merged**